src/qt/pivx/dashboardwidget.cpp

```diff
@@ -219,14 +220,17 @@ void DashboardWidget::loadWalletModel(){
         stakesFilter->setOnlyStakes(true);
         stakesFilter->setSourceModel(txModel);
         hasStakes = stakesFilter->rowCount() > 0;
-        loadChart();
+
+        onHideChartsChanged(GetBoolArg("-hidecharts", false));
```

Comment:
You can take the flag from the `optionModel` instead of re request it from the args.

Saying that, i'm not in favor of use `GetxxxArg` in any place outside of the init flow. Those are flags set once that will never change in the entire software lifecycle.

Reply:
Yes, good point. Updated.